app/assets/javascripts/comments.coffee

```diff
@@ -0,0 +1,20 @@
+class Comment
+  constructor: ->
+    @bindEvents()
+
+  bindEvents: =>
+    $(document).on "ajax:success", "[data-behavior='comment_destroy']",
```

Comment:
it can be more readable if you place all params at one line, or if you place every param at new line, dont mix pls